src/admin/localService/loadFile.js

```diff
@@ -0,0 +1,50 @@
+// @flow
+const fs = require("fs");
+const pathJoin = require("path").join;
+
+const readdir = fs.promises.readdir;
+
+/**
+ * load file into server memory
+ */
+async function loadFile(credDir: string, fileName: string): Promise<string> {
```

Comment:
This file is the same as the one in https://github.com/sourcecred/sourcecred/pull/1861/files, right? In that case, would be good to get that merged and rebase this on top. Once you have write access to the SC repo, next time this PR could just be merging into that other PR branch so we dont get duplicated diffs like this

Reply:
I sent @topocount an invite to join the sourcecred contributors team, which will give him write access